edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/brokerConnection/MqttBrokerProtocolHead.cs

```diff
@@ -73,7 +73,9 @@ public async Task CloseAsync(CancellationToken token)
             Events.Closed();
         }
 
-        public void Dispose() => this.CloseAsync(CancellationToken.None).Wait();
+        public void Dispose()
```

Comment:
Can you add a description on why we want to deviate from the dispose pattern?

Reply:
Just added to the description.